### PR TITLE
contribution process and change workflow

### PR DESCRIPTION
**Pull request by yodasw16**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* Unit tests
+* Spelling and grammar fixes
+
+Minor changes should begin with an [issue on Github](https://github.com/sparkdesignsystem/spark-design-system/issue) if you are unable to make the change yourself or are unsure if the change you want to make is needed. If you make the changes yourself, you can [open a pull request](https://github.com/sparkdesignsystem/spark-design-system/pulls) for review.
+
+If a minor change turns out to be substantial, the Spark team may ask you to go through the contribution workflow for substantial changes instead.
+
+### Substantial Changes
+
+Substantial generally changes fall into 2 categories: New components and amendments to an existing component that change it's design or behavior. This type of change requires you to complete our contribution workflow to determine if a new component or change to an existing component aligns with the direction Spark is going.
```
Comment (scottux) [inline CONTRIBUTING.md]: switch the words generally and changes here.

Comment (scottux) [reply CONTRIBUTING.md]: it's should be its.

File: src/pages/docs/contributions.hbs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <li>Performance improvements</li>
+      <li>Accessibility improvements</li>
+      <li>Unit tests</li>
+      <li>Spelling and grammar fixes</li>
+    </ul>
+    <p>Minor changes should begin with an <a href="https://github.com/sparkdesignsystem/spark-design-system/issue">issue on Github</a> if you are unable to make the change yourself or are unsure if the change you want to make is needed. If you make the changes yourself, you can <a href="https://github.com/sparkdesignsystem/spark-design-system/pulls">open a pull request</a> for review.</p>
+    <p>If a minor change turns out to be substantial, the Spark team may ask you to go through the contribution workflow for substantial changes instead.</p>
+
+    <h3 class="drizzle-b-h3">Substantial Changes</h3>
+    <p>Substantial generally changes fall into 2 categories: New components and amendments to an existing component that change it's design or behavior. This type of change requires you to complete our contribution workflow to determine if a new component or change to an existing component aligns with the direction Spark is going.</p>
```
Comment (scottux) [inline src/pages/docs/contributions.hbs]: same comment as above, "changes generally".

File: CONTRIBUTING.md (hunk 1 above)
Comment (scottux) [reply CONTRIBUTING.md]: it's should be its in this use case.

Comment (scottux) [reply CONTRIBUTING.md]: For the curious: "it's" should only be used for a contraction, _it is_ or _it has_, where "its" is the possessive like _hers_ or _theirs_.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* Unit tests
+* Spelling and grammar fixes
+
+Minor changes should begin with an [issue on Github](https://github.com/sparkdesignsystem/spark-design-system/issues) if you are unable to make the change yourself or are unsure if the change you want to make is needed. If you make the changes yourself, you can [open a pull request](https://github.com/sparkdesignsystem/spark-design-system/pulls) for review.
+
+If a minor change turns out to be substantial, the Spark team may ask you to go through the contribution workflow for substantial changes instead.
+
+### Substantial Changes
+
+Substantial changes generally fall into 2 categories: New components and amendments to an existing component that change its design or behavior. This type of change requires you to complete our contribution workflow to determine if a new component or change to an existing component aligns with the direction Spark is going.
```
Comment (afebbraro) [inline CONTRIBUTING.md]: 2 should be two

File: src/pages/docs/change-workflow.hbs [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+<div class="drizzle-o-Layout__content drizzle-b-LongFormText">
+  <div class="drizzle-o-ContentGrouping">
+    <p class="drizzle-b-Lede">This is the process the Spark Design System uses to research, design, build, and document Spark components.</p>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+
+    <div class="drizzle-o-ContentGrouping">
+      <h2 class="drizzle-b-h2">Change categories</h2>
+      <p>Every change to Spark will have tasks that fall into at least one of these categrories. Often a change will have tasks in many or all of them. The 4 categories are: <b>Research</b>, <b>Design</b>, <b>Build</b>, and <b>Document</b>.</p>
```
Comment (afebbraro) [inline src/pages/docs/change-workflow.hbs]: Some of these copy areas exceed the max line length. If you wouldn't mind, could you add some returns in?

Comment (afebbraro) [inline src/pages/docs/change-workflow.hbs]: 4 should be four

File: src/pages/docs/change-workflow.hbs [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        <li>Build issue titles must begin with [Build]</li>
+        <li>Code changes must follow the guidelines in <a class="drizzle-b-Link" href="/gettingstarted/developers">Getting Started - Developers</a> for code style.</li>
+        <li>Build tasks must follow the <a class="drizzle-b-Link" href="/docs/contributions">Contribution Guide</a>.</li>
+        <li>Code changes must be approved by a member of Spark's core team and should be reviewed by Spark's design lead when design or interaction are affected.</li>
+        <li>If a Document task is required and doesn't already exist, it should be created at this point.</li>
+      </ul>
+    </div>
+
+    <div class="drizzle-o-ContentGrouping">
+      <h4 class="drizzle-b-h4">Process</h4>
```
Comment (afebbraro) [inline src/pages/docs/change-workflow.hbs]: PROCESS doesn't should up as bold weight. It makes it not look like a heading when viewed with all the other content. What do you think?

File: src/pages/docs/change-workflow.hbs [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+  <div class="drizzle-o-ContentGrouping">
+
+    <div class="drizzle-o-ContentGrouping">
+      <h2 class="drizzle-b-h2">Change categories</h2>
+      <p>Every change to Spark will have tasks that fall into at least one of these categrories. Often a change will have tasks in many or all of them. The 4 categories are: <b>Research</b>, <b>Design</b>, <b>Build</b>, and <b>Document</b>.</p>
+    </div>
+
+    <div class="drizzle-o-ContentGrouping">
+      <h3 class="drizzle-b-h4">Issue titles</h3>
```
Comment (afebbraro) [inline src/pages/docs/change-workflow.hbs]: Same thing about the boldness and Issue-titles

File: src/pages/docs/change-workflow.hbs [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+      <p>We use Github issue titles to indicate which category an issue relates to. The format of issue titles looks like this:</p>
```
Comment (afebbraro) [inline src/pages/docs/change-workflow.hbs]: All uses of the word Github should be changed to GitHub.

**Pull request merged**